apps/editing-toolkit/editing-toolkit-plugin/premium-content/subscription-service/class-token-subscription-service.php

```diff
@@ -1,29 +1,39 @@
-<?php declare( strict_types = 1 );
+<?php
 /**
- * @package A8C\FSE\Earn
- *
  * A paywall that exchanges JWT tokens from WordPress.com to allow
  * a current visitor to view content that has been deemed "Premium content".
+ *
+ * @package A8C\FSE\Earn
  */
+
+declare( strict_types = 1 );
+
 namespace A8C\FSE\Earn\PremiumContent\SubscriptionService;
 
 use A8C\FSE\Earn\PremiumContent\JWT;
 
 // phpcs:ignore ImportDetection.Imports.RequireImports.Symbol
-abstract class Token_Subscription_Service implements Subscription_Service {
 
+/**
+ * Class Token_Subscription_Service
+ *
+ * @package A8C\FSE\Earn\PremiumContent\SubscriptionService
+ */
+abstract class Token_Subscription_Service implements Subscription_Service {
 
 	const JWT_AUTH_TOKEN_COOKIE_NAME = 'jp-premium-content-session';
 	const DECODE_EXCEPTION_FEATURE   = 'memberships';
 	const DECODE_EXCEPTION_MESSAGE   = 'Problem decoding provided token';
 	const REST_URL_ORIGIN            = 'https://subscribe.wordpress.com/';
 
 	/**
+	 * Initialize the token subscription service.
+	 *
 	 * @inheritDoc
 	 */
 	public function initialize() {
 		$token = $this->token_from_request();
-		if ( $token !== null ) {
+		if ( null !== $token ) {
```

Comment:
Is there a performance difference in reversing this? Or is it a linting error?

Reply:
The WordPress project has a preference for Yoda conditions, but other than that, I doubt it. 🤔

Reply:
Yup, this change was to satisfy our linter.